website/docs/d/managed_application_definition.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "Managed Application"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_application_definition"
+description: |-
+  Gets information about an existing Managed Application Definition
+---
+
+# Data Source: azurerm_managed_application_definition
+
+Uses this data source to access information about an existing Managed Application Definition.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_managed_application_definition" "example" {
+  name                = "example-managedappdef"
+  resource_group_name = "example-resources"
+}
+
+output "id" {
+  value = data.azurerm_managed_application_definition.example.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Managed Application Definition.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where this Managed Application Definition exists.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ID of the Managed Application Definition.
+
+* `location` - The Azure location where the resource exists.
```

Comment:
We shall put `location` in argument reference section.

Reply:
updated